src/main/java/com/google/sps/data/AvailabilityJSONConverter.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+public class AvailabilityJSONConverter {
```

Comment:
Nit: This class doesn't convert anything. I understand where the name came from, but as a rule I'd suggest sticking close to the most literal definition for what it is. By my reckoning, it's a `PutAvailabilityRequest`.

Etymology:

Put-Availability-Request
- Put: It's a `Put` request, and it will probably not be reused outside of the `Put` context. You're not going to `Get` using this.
- Availability: (Self explanatory)
- Request: Indicates to reader that this is directly coupled to REST HTTP api, and probably received by some servlet. You're probably not going to send something exactly like this back (as in, a Response).

Reply:
Done